austation/code/game/objects/items/robot/robot_items.dm

```diff
@@ -0,0 +1,82 @@
+/obj/item/borg/apparatus/food //basically beaker apparatus but for food items its janky but it works
+	name = "food manipulator"
+	desc = "A repurposed beaker apparatus for food items."
+	icon_state = "borg_beaker_apparatus"
+	storable = list(/obj/item/reagent_containers/food/snacks,
+	/obj/item/reagent_containers/food/condiment/flour,
+	/obj/item/reagent_containers/food/condiment/rice,
+	/obj/item/reagent_containers/food/condiment/soymilk,
+	/obj/item/reagent_containers/food/condiment/milk,
+	/obj/item/reagent_containers/food/condiment/mayonnaise,
+	/obj/item/reagent_containers/food/condiment/soysauce,
+	/obj/item/reagent_containers/food/condiment/sugar,
+	/obj/item/reagent_containers/food/condiment/peppermill,
+	/obj/item/reagent_containers/food/condiment/saltshaker) //yes this means you can do bread and i had to exempt enzymes so you dont pickup your internal one, which causes problems.

```

Comment:
you know you could just check if the object you're trying to grab is within your own body, right?

Reply:
I wouldnt know how to do that